shared/readline/readline.c

```diff
@@ -330,9 +330,9 @@ int readline_process_char(int c) {
         } else {
             rl.escape_seq = ESEQ_NONE;
             if (c == 'A') {
-#if MICROPY_REPL_EMACS_KEYS
-up_arrow_key:
-#endif
+                #if MICROPY_REPL_EMACS_KEYS
+            up_arrow_key:
+                #endif
```

Comment:
Curses!

Reply:
I'm tempted to skip changes to readline.c in favour of making a separate PR to clean up these warts. It feels a little out of scope to apply a manual rewrite here?

Reply:
@Gadgetoid That seems like a very good idea to me. If you're going to make that change then feel free to request a review after that and I'll take another pass over it. Thanks for perservering!